### PR TITLE
[Chore #150397532] Add branch settings

### PR DESCRIPTION
**Pull request by THEozmic**
I'm still trying to tweak the travis settings

### CODE REVIEW

File: travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
     - "6.11.0"
+branches:
+  only:
+      - staging
```
Comment (mradeybee) [inline travis.yml:6]: You should add master too.

**Review by mradeybee — CHANGES_REQUESTED** [review]

File: travis.yml (hunk 1 above)
Comment (THEozmic) [reply travis.yml:6]: Done

**Review by THEozmic — COMMENTED** [review]
I used --amend

**Review by mradeybee — APPROVED** [review]
👍 GTM

**Pull request merged**